src/EditController.js

```diff
@@ -30,16 +34,31 @@ class EditController extends ViewController {
     router.route("POST", routePrefix+"/edit/:id", ::this.save)
     router.route("POST", routePrefix+"/delete/:id", ::this.remove)
 
+    this.redirectAfterCreate = options.redirectAfterCreate || ""
+    this.redirectAfterEdit = options.redirectAfterEdit || ""
+    this.redirectAfterDelete = options.redirectAfterDelete || ""
+    
     // Yes, these should be __dirname not local to the subclass
     // Subclass templates are expected to be loaded by MVCModule or manually?
     templater.default().template(__dirname+"/templates/web-controller-form.ejs", this.pageTemplate, this.templatePrefix+"-create")
     templater.default().template(__dirname+"/templates/web-controller-form.ejs", this.pageTemplate, this.templatePrefix+"-edit")
     templater.default().template(__dirname+"/templates/web-controller-paginator.ejs")
   }
 
+  /*
```

Comment:
I like it, thanks. (Aside, I can't believe `Object.entries` is still in proposal!?)